superset-frontend/src/explore/components/ExploreViewContainer/index.jsx

```diff
@@ -541,7 +541,11 @@ function ExploreViewContainer(props) {
       .map(message => {
         const matchingLabels = controlsWithErrors
           .filter(control => control.validationErrors?.includes(message))
-          .map(control => control.label);
+          .map(control =>
+            typeof control.label === 'function'
```

Comment:
Probably a bigger change and not sure how feasible etc, but if a function we should call the func before setting the label that way the component doesn't need to be ware of this or any other type and instead "just render", but not sure how worthy that approach would be assuming it would require bigger changes